lang/cs/Org.Apache.REEF.Network/Group/Driver/Impl/GroupCommDriver.cs

```diff
@@ -51,9 +51,11 @@ public sealed class GroupCommDriver : IGroupCommDriver
         private readonly string _driverId;
         private readonly string _nameServerAddr;
         private readonly int _nameServerPort;
+        private readonly int _defaultGroupNumberOfTasks;
```

Comment:
In fault tolerant senario, communication group will be created again during the recovery. The code that create a new group will be in (or will be called from) IActiveContext handler and preferably be shared no matter it is first time or in recovery. If the Default group is created in the constructor, it will be always created. If we want to use NewCommunicationGroup() to create one we need in IActiveContext handler, the driver has to remove default one from the GroupCommunicationDriver that sounds not good. Make sense?